README.md

```diff
@@ -1,2 +1,72 @@
-# flock
-Twitter on Dgraph
+## Flock 
+
+Flock is twitter Graph model built on Dgraph. 
+
+
+Flock has two parts, 
+- [Mutation program](./main.go)
+
+Makes use of the tweets available from twitter developer API's and builds the Graph model on top of Dgraph. Here is the Graph schema of Flock, 
+
+![Schema](./schema.JPG)
+
+- [Query client](./client/main.go)
+  
+Runs interesting graph queries on the tweets data stored in Dgraph. 
+
+# Running Flock
+
+## Obtaining twitter credentials
+
+We need to create a twitter developer account and an app to be able to fetch stream of tweets using 
+their APIs. Let's start with how to create a twitter developer account.
+
+- Apply for a twitter developer account [here](https://developer.twitter.com/en/apply/user) and 
+  follow the instructions. The series of steps would end with your email verification.
+- Create a twitter app from [this link](https://developer.twitter.com/en/apps/create). 
+  All fields are not `required`.  
```

Comment:
not sure why `required` is highlighted. `not` should have been highlighted instead.